### ISSUE TITLE
Broken enb cache

### ISSUE THREAD

**Issue by Nodge** [enb-make/enb-stylus#121]
It seems to be a bug in `stylus` tech.
Cache methods `cacheFileList()` and `needRebuildFileList()` are used without cache key:

```js
    .saveCache(function (cache) {
        cache.cacheFileList(this._globalFiles);
    })
    .needRebuild(function (cache) {
        this._globalFiles = this._filenamesToFileList(this._globals);

        return cache.needRebuildFileList(this._globalFiles);
    })
```

In this case enb rebuilds css everytime.

To fix the issue just add a cache key:
```js
    .saveCache(function (cache) {
        cache.cacheFileList('stylus', this._globalFiles);
    })
    .needRebuild(function (cache) {
        this._globalFiles = this._filenamesToFileList(this._globals);

        return cache.needRebuildFileList('stylus', this._globalFiles);
    })
```

**Comment by tadatuta** [comment]
@levonet please, consider this issue for nearest sprint

**Comment by blond** [comment]
Fixed in #122
